pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jboss</groupId>
         <artifactId>jboss-parent</artifactId>
-        <version>21</version>
+        <version>11</version>
```

Comment:
That version is not available for EAP 7.0. For now I'm going to revert to 11 and revisit once we get to actually needing it. In other words, once we actually have a properly filed issue that outlines any Java 9 problems with the current build.